scripts/vcpkg_ports/umpire/v2024.07.0-chrono-include.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/umpire/event/event.hpp b/src/umpire/event/event.hpp
+index 03f0d37..db84c1c 100644
+--- a/src/umpire/event/event.hpp
++++ b/src/umpire/event/event.hpp
+@@ -7,6 +7,7 @@
+ #ifndef UMPIRE_event_HPP
+ #define UMPIRE_event_HPP
+ 
++#include <chrono>
```

Comment:
@davidbeckingsale -- My local machine did not need this on Windows, but our CI needed this for umpire@2024.07.0.
I saw that it was recently added to umpire@develop -- https://github.com/LLNL/Umpire/pull/934